src/Form/Form.js

```diff
@@ -375,14 +375,29 @@ class Form {
         const mainInputType = getInputMainType(input)
         this.inputs[mainInputType].add(input)
 
+        // TODO: temporary pixel
+        const subtype = getInputSubtype(input)
+        if (subtype === 'emailAddress') {
+            this.addListener(input, 'pointerdown', () => {
+                this.device.firePixel({pixelName: 'incontext_eligible'})
+            }, {once: true})
+        }
```

Comment:
Fired once per page. This is gated appropriately on the client side to keep the implementation simpler. It's a temporary pixel anyway.

Reply:
`decorateInput` is rather convoluted and has a ton of stuff in it. Adding it here makes it easy to add/remove. Given that this is a temporary thing it makes sense to leave it here.